src/Util.hs

```diff
@@ -27,5 +28,14 @@ menusToEitherNoLunch = \case
 -- | Remove text tags that only contain whitespace.
 removeWhitespaceTags :: [Tag ByteString] -> [Tag ByteString]
 removeWhitespaceTags =
-  filter (\t -> not (isTagText t) || tagText (not . BL.all W8.isSpace) t)
-
+  filter (or . ([not . isTagText, tagText (not . BL.all W8.isSpace)] <*>) . pure)
+
+-- |  Increase the date by one with and handles overflow
+--    I cannot believe that this isn't in the library
+nextDay :: YearMonthDay -> YearMonthDay
+nextDay (YearMonthDay y m d)
+  | d < len   = YearMonthDay  y       m      (d + 1)
+  | m < 12    = YearMonthDay  y      (m + 1) (d - len)
+  | otherwise = YearMonthDay (y + 1)  1      (d - len)
```

Comment:
Couldn't these just set the day to 0?
```suggestion
  | m < 12    = YearMonthDay  y      (m + 1)  0
  | otherwise = YearMonthDay (y + 1)  1       0
```

Reply:
Right, these should be one but since they are bounded they will bounce up to being a 1

Reply:
Ohh I was wondering why they were using different bases for months and days, that explains it